ComponentLibWithXmlLinkerConfig/ComponentLibWithXmlLinkerConfig.csproj

```
@@ -5,6 +5,16 @@
     <RazorLangVersion>3.0</RazorLangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="ComponentLibWithXmlLinkerConfig.xml" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="ComponentLibWithXmlLinkerConfig.xml">
+      <LogicalName>ComponentLibWithXmlLinkerConfig.xml</LogicalName>
```

I haven't tested this, but it's worth trying:

```suggestion
      <LogicalName>$(MSBuildProjectName).xml</LogicalName>
```

If that works, it would simplify any docs, as we wouldn't need to tell people to give the file a particular special name. You could then change the name of the file to just `LinkerConfig.xml`.

---

Works for me.